shared/js/manifest_helper.js

```diff
@@ -51,6 +51,10 @@ function ManifestHelper(manifest) {
   }
 }
 
+ManifestHelper.prototype.getShortNameOrName = function() {
```

Comment:
Hmm, I'm not sure. I was intentionally very unambiguous because this is needed for security reasons. Not sure what you want to be forwards compatible with?